dom-event-delegation/main.js

```diff
@@ -0,0 +1,16 @@
+var $taskList = document.querySelector('.task-list');
+
+$taskList.addEventListener('click', handleClick);
+
+function handleClick(event) {
+  console.log('event.target:', event.target);
+  console.log('event.target.tagName:', event.target.tagName);
+  var closestTaskList = event.target.closest('.task-list-item');
+  console.log('closest .task-list-item', closestTaskList);
+  closestTaskList.remove();
+
+  if (event.target.tagName !== 'BUTTON') {
+    // eslint-disable-next-line no-useless-return
+    return;
+  }
```

Comment:
make sure you perform this check before the closest element is found

Reply:
ah, that's what I was going to ask about. lol. Let me re-check